vsintegration/src/FSharp.LanguageService/ProjectSitesAndFiles.fs

```diff
@@ -182,7 +184,7 @@ type internal ProjectSitesAndFiles() =
              UseScriptResolutionRules = SourceFile.MustBeSingleFileProject fileName
              LoadTime = projectSite.LoadTime
              UnresolvedReferences = None
-             OriginalLoadReferences = []
+             OriginalLoadReferences =  (projectSite.AssemblyReferences() |> Array.map(fun s -> rangeCmdArgs, s) |> Array.toList)
```

Comment:
This doesn't look right. ``OriginalLoadReferences`` doesn't contain assembly references - it contains ``#load`` references.  It should be renamed to ``OriginalHashLoadDIrectives`` or something.  In more detail:

* The comment on the declaration of ``OriginalLoadReferences`` is ``Unused in this API``.  
* In reality, it seems it is populated only for scripts, i.e. ``OriginalLoadReferences = loadClosure.OriginalLoadReferences``.  Its purpose is to make sure the incremental builder for script checking gets re-created if the ``#load`` change in any way.
* You seem to be trying to reuse it to propagate the assembly references some how, including for projects, I'm not quite sure how or if it's essential to the PR

Reply:
@dsyme yes, I agree ... it is from when I first did this, I will fix it before I commit.

Reply:
That's ok, just document it please :)

Reply:
No ... I will remove it, it is not necessary.  And I was trying to do exactly what you thought.  references are currently propagated using the command line options ... for better or worse.